modules/microshift-nw-ipv6-concept.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-intro-ipv6_{context}"]
+= IPv6 networking with {microshift-short}
+
+The {microshift-short} service defaults to IPv4 address families. However, IPv6 single-stack and IPv4/IPv6 dual-stack networking is available on supported platforms. When you set the values for IPv6 in the {microshift-short} configuration file and restart the service, settings managed by the OVN-Kubernetes network plugin are updated automatically. After migrating to dual-stack networking, both new and existing pods have dual-stack networking enabled.
```

Comment:
There might be a note here. When migrating to dual stack existing pods need to be restarted or else they remain with a single ip family. The same applies to services.